src/connection/stream.rs

```diff
@@ -131,16 +131,19 @@ impl Stream {
         self.id
     }
 
+    pub fn is_write_closed(&self) -> bool {
```

Comment:
Is this a breaking change? If so, this would require a changelog entry.

Reply:
Adding a new function is not a breaking change I believe!

Reply:
Silly me. Sorry for the noise.